src/js/modules/infragistics.util.js

```diff
@@ -29,7 +29,7 @@
 	} else {
 
 		// Browser globals
-		factory();
+		return factory();
```

Comment:
Here the return is OK, but remove the `// Browser globals` comment, since the module actually returns `igRoot`.